.github/workflows/accessibility.yml

```diff
@@ -1,5 +1,11 @@
 name: accessibility
-on: [push]
+on:
+  push:
+    branches:
+      - 3.2
+  pull_request:
+    branches:
+      - 3.2
```

Comment:
```suggestion
```

Reply:
This should be probably the same as the main cypress workflow. Running these on a PR won't detect any new issues anyway. As you know we're running tests against a snap build, not latest maas-ui source code.

Reply:
Thanks @petermakowski!